packages/core/ios/Fabric/LottieAnimationViewComponentView.h

```diff
@@ -0,0 +1,26 @@
+#ifdef __cplusplus
```

Comment:
prevents issue `lottie-react-native/apps/fabric/ios/Pods/Headers/Public/React-Fabric/react/renderer/core/EventQueue.h:10:10: 'memory' file not found`

Reply:
Did you find that in any specific documentation? Just in case there were more information around that specific line - I would be keen to learn a little more about that 😃

Reply:
No documentation. I had a similar issue a few months ago with Expo modules on the new arch and the Expo team helped me figure this out.